docs/quick-start.md

```diff
@@ -11,7 +11,7 @@ This guide assumes you are using MacOS or a Linux machine and are an Automattici
 ### Using the installation script
 
 To speed up the installation process, you may use our monorepo installation script. To do so:
- - clone the Jetpack repo: `git clone https://github.com/Automattic/jetpack.git`
+ - clone the Jetpack repo: `git clone git@github.com:Automattic/Jetpack.git` using a public SSH key ([recommended](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#clone-the-repository)) or `git clone https://github.com/Automattic/jetpack.git` using HTTPS
```

Comment:
What do you think about cleaning this up a bit, maybe like this?

```suggestion
 - clone the Jetpack repo using one of these two methods:
 	- A public SSH key ([recommended](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#clone-the-repository)): `git clone git@github.com:Automattic/Jetpack.git`
 	-  HTTPS: `git clone https://github.com/Automattic/jetpack.git`
```

Reply:
Love it, will add that.